code/controllers/stonedmc/subsystem/chat.dm

```diff
@@ -0,0 +1,69 @@
+SUBSYSTEM_DEF(chat)
+	name = "Chat"
+	flags = SS_TICKER|SS_NO_INIT
+	wait = 1
+	priority = FIRE_PRIORITY_CHAT
+
+	var/list/payload = list()
+
+
+/datum/controller/subsystem/chat/fire()
+	for(var/i in payload)
+		var/client/C = i
+		C << output(payload[C], "browseroutput:output")
+		payload -= C
+
+		if(MC_TICK_CHECK)
+			return
+
+
+/datum/controller/subsystem/chat/proc/queue(target, message, handle_whitespace = TRUE)
+	if(!target || !message)
+		return
+
+	//Sorry but this will have to do for a while.
+	if(!istext(message))
+		if(istype(message, /image) || istype(message, /sound) || istype(target, /savefile) || !ismob(target))
```

Comment:
```suggestion
		if(istype(message, /image) || istype(message, /sound) || !ismob(target))
```
Same logic as before. If it's not a mob, it can be a save file.